TokenBucket.php

```diff
@@ -44,8 +44,7 @@ public function __construct($key, Backend $backend, TokenRate $rate) {
 
    /**
     * Tries to consume a token, if a token can't be consumed, then the interval
-    * of seconds for when the consume will be valid will be.
-    * is returned for when the next token will be available.
+    * of seconds for when the consume will be valid.
```

Comment:
Seems like a broken sentence.

How about:
Tries to consume a given number of tokens. Returns `[$success, $time]` where `$time` is the number of seconds until the specified number tokens will be available.